app/src/main/java/org/stepic/droid/ui/fragments/SectionsFragment.java

```diff
@@ -858,36 +845,14 @@ public void updatePosition(int position) {
 
     @Override
     public void onShowInvitationDialog(@NotNull final Course courseForSharing) {
-        SpannableString inviteTitle = new SpannableString(getString(R.string.take_course_with_fiends));
-        inviteTitle.setSpan(new ForegroundColorSpan(Color.BLACK), 0, inviteTitle.length(), Spannable.SPAN_EXCLUSIVE_EXCLUSIVE);
-        CalligraphyTypefaceSpan typefaceSpan = new CalligraphyTypefaceSpan(TypefaceUtils.load(getContext().getAssets(), getFontsProvider().provideFontPath(FontType.bold)));
-        inviteTitle.setSpan(typefaceSpan, 0, inviteTitle.length(), Spannable.SPAN_EXCLUSIVE_EXCLUSIVE);
-
-
-        MaterialStyledDialog dialog = new MaterialStyledDialog.Builder(getContext())
-                .setTitle(inviteTitle)
-                .setDescription(R.string.invite_friends_description)
-                .setHeaderDrawable(R.drawable.dialog_background)
-                .setPositiveText(R.string.invite)
-                .setNegativeText(R.string.dont_want)
-                .setScrollable(true, 10) // number of lines
-                .onPositive(new MaterialDialog.SingleButtonCallback() {
-                    @Override
-                    public void onClick(@NonNull MaterialDialog dialog, @NonNull DialogAction which) {
-                        getAnalytic().reportEvent(Analytic.Interaction.POSITIVE_MATERIAL_DIALOG_INVITATION);
-                        Intent intent = shareHelper.getIntentForCourseSharing(courseForSharing);
-                        SectionsFragment.this.startActivityForResult(intent, INVITE_REQUEST_CODE);
-                    }
-                })
-                .onNegative(new MaterialDialog.SingleButtonCallback() {
-                    @Override
-                    public void onClick(@NonNull MaterialDialog dialog, @NonNull DialogAction which) {
-                        invitationPresenter.onClickDecline();
-                        showMessageAboutSharing();
-                    }
-                })
-                .build();
-        dialog.show();
+        rootView.postDelayed(new Runnable() {
+            @Override
+            public void run() {
+                final View shareView = rootView.findViewById(R.id.menu_item_share);
```

Comment:
`onCreateOptionsMenu` calling order is not defined (https://stackoverflow.com/a/17727854/5222184) and can be after calling `onShowInvitationDialog`. So, we should develop other solution.

Reply:
Although, I think it will never affect app in practice. (we use INVITE_FRIEND_POPUP_DELAY)
So let's log to analytic, when shareView is null. Report analytic event `invite_dialog_broken`